content/en/docs/concepts/architecture/nodes.md

```diff
@@ -300,15 +298,15 @@ If you want to explicitly reserve resources for non-Pod processes, follow this t
 If you have enabled the `TopologyManager`
 [feature gate](/docs/reference/command-line-tools-reference/feature-gates/), then
 the kubelet can use topology hints when making resource assignment decisions.
-
-## API Object
-
-Node is a top-level resource in the Kubernetes REST API. More details about the
-API object can be found at:
-[Node API object](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#node-v1-core).
+See [Control Topology Management Policies on a Node](/docs/tasks/administer-cluster/topology-manager/)
+for more information.
 
 {{% /capture %}}
 {{% capture whatsnext %}}
-* Read about [node components](/docs/concepts/overview/components/#node-components)
-* Read about node-level topology: [Control Topology Management Policies on a node](/docs/tasks/administer-cluster/topology-manager/)
+* Learn about the [components](/docs/concepts/overview/components/#node-components) that make up a node.
+* Read the [API definition for Node](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#node-v1-core)
```

Comment:
nit: No period at end of list items?

Reply:
The fix is to add a period after each item?

Reply:
This is a nit. The fix is to be consistent.